core/src/main/java/io/undertow/server/RequestStatistics.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.undertow.server;
+
+public final class RequestStatistics {
+    private final HttpServerExchange exchange;
+    private final long startTime = System.nanoTime();
+
+    public RequestStatistics(HttpServerExchange exchange) {
+        this.exchange = exchange;
+    }
+
+    public long getBytesSent() {
+        return exchange.getResponseBytesSent();
+    }
+
+    public long getBytesReceived() {
+        return exchange.getRequestContentLength();
+    }
+
+    public long getStartTime() {
+        return startTime;
```

Comment:
I think this can be changed to just return `exchange.getRequestStartTime()` and the `startTime` variable is no longer needed.

Reply:
Done